csharp/test/Drivers/Apache/Spark/DriverTests.cs

```diff
@@ -505,14 +510,71 @@ public async Task CanGetTableTypes()
         [SkippableFact, Order(10)]
         public void CanExecuteQuery()
         {
-            using AdbcConnection adbcConnection = NewConnection();
+            long count = 0;
+            long target = 80;
+            Queue<String> log = new Queue<string>();
+            string FileName = $@"c:\temp\hmsmeta\{DateTime.Now.ToString("yyyyMMdd_HHmm")}__{target}.log";
 
-            using AdbcStatement statement = adbcConnection.CreateStatement();
-            statement.SqlQuery = TestConfiguration.Query;
-
-            QueryResult queryResult = statement.ExecuteQuery();
+            for (int i = 0; i < target; i++)
```

Comment:
This test could be refactored to handle any test "kernel" (lambda) to wrap the test such that we could capture performance information in a log file. What do you think?